docs/installation/service.mdx

```diff
@@ -0,0 +1,118 @@
+---
+sidebar_position: 6
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Service
+
+## EVCC als Service
```

Comment:
```suggestion
## evcc als Service
```
Wir schreiben das inzwischen immer klein (wie im Logo).

Reply:
Ich habe dies in den 3 Files angepasst => evcc